package.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
If `permit-helper.js` is meant to be used by third-parties I would include it in the `index.js`